packages/alfa-rules/src/sia-r70/rule.ts

```diff
@@ -0,0 +1,147 @@
+/// <reference lib="dom" />
+
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Document, Element, Namespace } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { List } from "@siteimprove/alfa-list";
+import { Iterable } from "@siteimprove/alfa-iterable";
+
+import { expectation } from "../common/expectation";
+import { isIgnored, hasChild } from "../common/predicate";
+import { Group } from "../common/group";
+
+import { isRendered } from "../common/predicate/is-rendered";
+
+import { isDocumentElement } from "../common/predicate/is-document-element";
+
+const { isElement, hasName, hasNamespace } = Element;
+const { and, or, test, not } = Predicate;
+const deprecated = [
+  "applet",
+  "basefont",
+  "bgsound",
+  "big",
+  "blink",
+  "center",
+  "content",
+  "dir",
+  "font",
+  "frame",
+  "frameset",
+  "hgroup",
+  "image",
+  "keygen",
+  "marquee",
+  "menuitem",
+  "nobr",
+  "noembed",
+  "noframes",
+  "plaintext",
+  "rb",
+  "rtc",
+  "shadow",
+  "spacer",
+  "strike",
+  "tt",
+  "xmp",
+];
+export default Rule.Atomic.of<Page, Document, Group<Element>>({
+  uri: "https://alfa.siteimprove.com/rules/sia-r70",
+  evaluate({ device, document }) {
+    return {
+      applicability() {
+        return test(hasChild(isDocumentElement), document) ? [document] : [];
+      },
+
+      expectations(target) {
+        const deprecatedElements = target
+          .descendants({ flattened: true, nested: true })
+          .filter(isElement)
+          .filter(
+            and(
+              hasNamespace(Namespace.HTML),
+              hasName("acronym", ...deprecated),
+              isRendered(device)
+            )
+          );
+
+        //console.dir(deprecatedElements.toJSON(), { depth: null });
+
+        return {
+          1: expectation(
+            deprecatedElements.isEmpty(),
+            () => Outcomes.IsNotDeprecated,
+            () => Outcomes.IsDeprecated(deprecatedElements)
+          ),
+        };
+      },
+    };
+  },
+});
+
+export namespace Outcomes {
+  export const IsNotDeprecated = Ok.of(
+    Diagnostic.of(`The document doesn't contain deprecated elements`)
+  );
+
+  export const IsDeprecated = (errors: Iterable<Element>) =>
+    Err.of(
+      DeprecatedElements.of(
+        `The document contains deprecated elements: `,
+        errors
+      )
+    );
+}
+
+class DeprecatedElements extends Diagnostic implements Iterable<Element> {
+  public static of(
+    message: string,
+    errors: Iterable<Element> = []
```

Comment:
```suggestion
    errors: Iterable<Element> = List.empty()
```
Since we're using `List`, we can as well initialise with one.